releasenotes/notes/annotated-params-116288d5628f7ee8.yaml

```
@@ -0,0 +1,23 @@
+---
+features_circuits:
+  - |
+    Add support for :meth:`.AnnotatedOperation.params`, which enables
```

should the reno include the other new method too?

```suggestion
    Added support for :meth:`.AnnotatedOperation.params` and :meth:`.AnnotatedOperation.validate_param`, which enable
```